packages/shared/ReactDOMTypes.js

```diff
@@ -30,7 +30,7 @@ export type ReactDOMResponderEvent = {
   passiveSupported: boolean,
   pointerId: null | number,
   pointerType: PointerType,
-  responderTarget: null | Element | Document,
+  responderTarget: Element,
```

Comment:
All `responderTarget` should be `Element` type, and they are, but the previous Flow type meant all the responders needed to account for the other types.